.circleci/config.yml

```
@@ -0,0 +1,17 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: trussworks/circleci-docker-primary:4a25f819d871cefc0ab71b09c2598990208a2482
```

Use latest container.
```suggestion
      - image: trussworks/circleci-docker-primary:5ca2def958c1a6abbb6ac7668c13ecee5b4598e1
```